sessions/metrics.py

```diff
@@ -70,8 +72,12 @@ def update(self):
                         print('%s: %d' % (username, n))
                     self.g_sessions.labels(username).set(n)
                     self.lastusers.add(username)
+            except omero.CmdError, ce:
```

Comment:
Can you use `as ce` instead for python 3 compatibility?

Reply:
Force pushed.